src/views/Receiving.vue

```diff
@@ -70,6 +73,9 @@ export default defineComponent({
       }
       await this.store.dispatch("shipment/findShipment", payload);
     },
+    loadMoreShipments(){
+      this.getShipments(process.env.VUE_APP_VIEW_SIZE ,++this.viewIndex)
```

Comment:
We won't increment viewIndex in this way, refer [inventory count](https://github.com/hotwax/inventorycount/blob/main/src/views/Search.vue#L97) to refer how to implement an infinite scroll.

Reply:
Implemented this functionality as per your suggestion.